src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -1418,6 +1418,34 @@ public static void enableTranslogFlush(String index) {
         client().admin().indices().prepareUpdateSettings(index).setSettings(settings).get();
     }
 
+    /** Disables an index block for the specified index */
+    public static void disableIndexBlock(String index, String block) {
+        disableIndexBlock(client(), index, block);
+    }
+
+    /** Disables an index block for the specified index using a given {@link Client} */
+    public static void disableIndexBlock(Client client, String index, String block) {
```

Comment:
is it important to provide a specific client here? isn't the previous method enough?